example/test_problem/Hydro/CCSN/Input__TestProb.CoreCollapse

```diff
@@ -20,15 +20,24 @@ CCSN_Eint_Mode   1                                 # Mode of obtaining internal
-CCSN_CC_MaxRefine_Dens1   1.0e11                   # central density threshold that reduces the maximum refinement level 1 [1.0e11]
-CCSN_CC_MaxRefine_Dens2   1.0e12                   # central density threshold that reduces the maximum refinement level 2 [1.0e12]
+CCSN_CC_MaxRefine_Flag1   1                        # enable limiting maximum refinement level 1  [0]
+CCSN_CC_MaxRefine_Flag2   1                        # enable limiting maximum refinement level 2  [0]
+CCSN_CC_MaxRefine_LV1     6                        # reduced maximum refinement level 1 to this value  [MAX_LEVEL-2]
+CCSN_CC_MaxRefine_LV2     7                        # reduced maximum refinement level 2 to this value  [MAX_LEVEL-1]
+CCSN_CC_MaxRefine_Dens1   1.0e11                   # central density threshold that reduces the maximum refinement level 1  [1.0e11]
+CCSN_CC_MaxRefine_Dens2   1.0e12                   # central density threshold that reduces the maximum refinement level 2  [1.0e12]
 CCSN_MaxRefine_RadFac     0.15                     # factor that determines the maximum refinement level based on distance from the box center  [0.15]
 CCSN_CC_CentralDensFac    1.0e13                   # factor that reduces the dt constrained by the central density (in cgs) during the core collapse  [1.0e13]
-CCSN_CC_Red_DT            1.0e-5                   # reduced time step (in s) when the central density exceeds CCSN_CC_CentralDensFac before bounce [1e-5]
+CCSN_CC_Red_DT            1.0e-5                   # reduced time step (in s) when the central density exceeds CCSN_CC_CentralDensFac before bounce  [1e-5]
 CCSN_LB_TimeFac           0.10                     # factor that scales the dt constrained by lightbulb scheme  [0.1]
 
-CCSN_Is_PostBounce        0                        # boolean that indicates whether core bounce has occurred   [0]
+CCSN_CC_Form_Rot          0                        # add formulated shell-type initial angular velocity                            [0]
+                                                   # ( Omega(r)=Omega_0*[R_0^2/(r^2+R_0^2)], r: spherical radius )  (0=off, 1=on)  [0]
+CCSN_CC_Form_Rot_R_0      1.03376e8                # rotational parameter R_0 (in cm) in the rotatinal profile
+                                                   # ( Omega(r)=Omega_0*[R_0^2/(r^2+R_0^2)], r: spherical radius )                 [2.0e8]
+CCSN_CC_Form_Rot_Omega_0  2.0                      # rotational parameter Omega_0 (in rad/s) in the rotatinal profile
+                                                   # ( Omega(r)=Omega_0*[R_0^2/(r^2+R_0^2)], r: spherical radius )                 [0.5]
+CCSN_CC_Rot_Mag           0                        # flag for magnifying rotational velocity by constant, CCSN_CC_Rot_Mag_Fact     [0]
+CCSN_CC_Rot_Mag_Fact      1.0                      # magnification factor for rotational veolocity                                 [1.0]
```

Comment:
- typos
  - I corrected them.

- On `CCSN_CC_Form_Rot*`
  - I added some physical meanings for the parameters.
  - I changed the parameters' names.
  - There is no option like that. We either add a rotational profile from either formula for IC.
    but, for the IC, if the IC doesn't have the rotation `omega=0.0` would be recorded.
    My approach is that if there is any rotational velocity in `IC`. It is consistent with using that initial velocity.
    Or just in case, we want to add an artificial rotation profile, we can use `CCSN_CC_Form_Rot*`
  - I removed the prefix.
  - Not so meaningful. I am following this [paper](https://iopscience.iop.org/article/10.1088/0004-637X/793/1/45/pdf).
  and the moderate rotational model here has `Omega_0 = 0.1 pi ~ 0.5`. `Omega_0=1` also doesn't have any physical meaning.
  
- On `CCSN_CC_Rot_Mag_*`
  - I changed it to `Amp`
  - No. I don't think so, we can either increase or decrease the rotational rate.
  - That's a good idea. I merged it.

Reply:
- On `CCSN_CC_Form_Rot*`
  - is `Omega0` the "rotational velocity"?
  - so what should I do if I do not want any rotational profile from either formula or IC? And I want to save the time for initializing the rotational velocity?
- On `CCSN_CC_Rot_Mag_*`
  - Since the value of `CCSN_CC_Rot_Amp` can be less than 1, both the name `Mag` and `Amp` are misleading.

Reply:
- On `CCSN_CC_Form_Rot*`
  - To be exact, `Omega0` is the rotational velocity (speed) at the center.
  - For now, if there is rotation in `IC`. No way not to use it. Do you think we need this feature even if it is not consistent with other quantities in `IC`. If we need this feature, then we can add it.
- I am not so sure this is as misleading as that, We can say `0.5 times` or something like that. If you think this is that much misleading do you have some suggestions?

Reply:
- On `CCSN_CC_Form_Rot*`
  - Ok, my bad. Both rotational velocity and angular velocity refer to the same thing. Then, use only one of them in describing Omega in this PR
  - To me, I think it is necessary. And it is NOT DIFFICULT to add this feature. 
- On `CCSN_CC_Rot_Mag_*`
  - Dictionary of [amplify](https://www.merriam-webster.com/dictionary/amplify) and [magnify](https://www.merriam-webster.com/dictionary/magnify). Do you think you amplify or magnify the rotational profile if `CCSN_CC_Rot_[Mag/Amp] < 1`?

Reply:
I also added whether we are going to add rotation or not in any cases.
I don't think this is serious, but anyway, I have changed the parameter name `CCSN_CC_Rot_Amp` to `CCSN_CC_Rot_Fac`

Reply:
- I did not get the point why we need to enable the rotational profile by default.
- Do you consider to merge `CCSN_Is_Rotation` and `CCSN_CC_Rot`? Say `CCSN_CC_Rot =0` for no additional rotational profile, `1` for formulated profile, and `2` for tabulated profile in IC?
- Fine, I will not comment anymore on the naming/typos issue for your PR.

Reply:
- The reason why I enable the rotational profile by default is that, if we have `IC` with initial angular velocity then it is self-consistent to use the rotational profile by default not just discard it.

- Actually, I considered this and somehow I thought this is not good, but looked back this is a good method. So I merged them.

- I didn't say you shouldn't comment on naming/typos. I thought this specific comment was not that meaningful.